modules/hivemq/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,22 @@
+<configuration scan="false">
+
+    <appender name="CONSOLE" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>%-30(%d %level)- %msg%n%ex</pattern>
+        </encoder>
+    </appender>
+
+    <root>
+        <appender-ref ref="CONSOLE"/>
+    </root>
+
+    <root level="DEBUG">
+        <appender-ref ref="CONSOLE"/>
+    </root>
+
+    <logger name="com.hivemq.client" level="OFF"/>
+    <logger name="org.testcontainers" level="OFF"/>
```

Comment:
Turning Testcontainers and HiveMQ logs OFF here seems like it would impede debugging of any problems that occur later. Is it left over from something?

Reply:
Yes this is a leftover. It is kind of related to: https://github.com/testcontainers/testcontainers-java/pull/4797#discussion_r783951593